generators/app/index.js

```diff
@@ -18,6 +18,11 @@ module.exports = Generator.extend({
     return this.prompt(prompts).then(function (options) {
       // To access options later use this.options.someAnswer;
       this.options = options;
+
+      // Compose
+      this.composeWith('one-base:gulp', {
```

Comment:
When I updated to 1.0 the other day I tried this and couldn't get it to work, but my syntax could have been off. I'll try again.